### PR TITLE
JBPM-6793 - Remove onclick and onmouseover in process diagram svg con…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…tent

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac please take a look.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java [hunk 1]
```diff
@@ -61,29 +60,27 @@ public void onProcessInstanceSelectionEvent(@Observes ProcessInstanceSelectionEv
...
+        processImageService.call((String svgContent) -> displayImage(svgContent)).getProcessDiagram(serverTemplateId,
+                                                                                                    containerId,
+                                                                                                    processId);
+    }
+
+    protected void displayImage(final String svgContent) {
+        if (svgContent == null || svgContent.isEmpty()) {
+            designerWidget.displayMessage(constants.Process_Diagram_Not_Found());
+        } else {
+            designerWidget.displayImage(svgContent);
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java]: Can we add a test or two for this logic?

**Review by bsremac — COMMENTED** [review]
As simple as it gets.
Just one comment.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java]: done

**Review by bsremac — APPROVED** [review]

**Pull request merged**